shared-bindings/wifi/Radio.c

```diff
@@ -304,11 +304,11 @@ STATIC mp_obj_t wifi_radio_stop_ap(mp_obj_t self) {
 MP_DEFINE_CONST_FUN_OBJ_1(wifi_radio_stop_ap_obj, wifi_radio_stop_ap);
 
 //|     def connect(self,
-//|                 ssid: ReadableBuffer,
-//|                 password: ReadableBuffer = b"",
+//|                 ssid: str,
+//|                 password: str = b"",
```

Comment:
The given default value is a `bytes`, not a `str`.
```suggestion
//|                 password: str = "",
```

Reply:
SSID can be composed of any byte values, not only those that can be represented in a string.